javascript/chronometer.js

```diff
@@ -1,26 +1,51 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = 0;
+    this.running = false;
   }
+
   startClick(callback) {
-    // ... your code goes here
-  }
+    if (this.running) {
+      return;
+    }
+
+    this.running = true;
+
+    this.intervalId = setInterval(() => {
+      this.currentTime += 1;
+
+      if (callback) {
+        callback();
+      }
+    }, 1000);
+  }
+
   getMinutes() {
-    // ... your code goes here
+    return Math.floor(this.currentTime / 60);
   }
+
   getSeconds() {
-    // ... your code goes here
+    return this.currentTime - (Math.floor(this.currentTime / 60) * 60);
```

Comment:
You can get a simpler solution by using the [modulo operator](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/First_steps/Math#Arithmetic_operators)
```
return this.currentTime % 60;